src/shared-components/field/index.js

```diff
@@ -72,20 +69,14 @@ class Field extends React.Component {
           {React.cloneElement(inputChild, {
             disabled,
           })}
-
-          <TransitionGroup component={HelperList}>
-            {!!hintMessage && (
+          {!!hintMessage && (
+            <TransitionGroup component={HintList}>
```

Comment:
![field](https://user-images.githubusercontent.com/38407520/55914152-4956e200-5b9b-11e9-808f-dfd412d820ad.gif)

looks like the hint is acting up. I feel like we're mixing transitions with emotion and transitiongroup on this. The errors are being added/removed as they are validated, but if there is a hint message it mounts on the DOM and stays there, only the styling changes on focus.

Reply:
https://github.com/PocketDerm/radiance-ui/blob/master/src/shared-components/field/style.js#L46
^^ I think if we added max-height property here you could strip out the TransitionGroup and HelperTransition all together for the hint item

Reply:
that actually troubled my a lot. Since I wanted to reuse BulkErrors inside the Field I had to create 2 separate lists. Problem is Ben wanted to not show the push down layout that happens when you add messages at the bottom so thats why I kept a min-height at all times on both lists. You re right I actually didnt need Helpertransition for the Hint so I refactored it and simplified that HintItem.

I checked the code in CUrology and Hint Message is not used at all. So I just removed the min-heihgt from the Hint. Only the error list will have the min-heihgt that prevents the first error to push down the layout which is 90% of the cases.